recipes/tmap/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "tmap" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reymond-group/tmap/archive/9aa043dd1af022161048ee0a4b5554ec5e34977f.zip
+  sha256: 56cf4f83d663950ab7ef41deb425b8e1c01de378fb5239aeef7b6613a6edf212
+
+  patches:
+    - 01_bind2d_deprecated.diff
+    - 02_setup_windows.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+  binary_relocation: False  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - setuptools
+    - pybind11
+    - ogdf ==2020.02
+  host:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - python
+    - pip
+    - ogdf ==2020.02
+  run:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - python
+    - ogdf ==2020.02
```

Comment:
Dose this need to be pinned so tightly?

Reply:
Releases look very sparsed (https://github.com/ogdf/ogdf/releases) so I thought that would make sense to fix it. I can remove it if you prefer. Let me know.

Reply:
You can now remove this dep entirely because of the run export.

Reply:
> You can now remove this dep entirely because of the run export.

Not the one in `requirements: host` ;).

> Releases look very sparsed 

Correct, OGDF is not released very often at all and on a as-needed basis. I'd recommend pinning it do `2020.20`.